Hippo.Web/Startup.cs

```diff
@@ -186,9 +186,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<INotificationService, NotificationService>();
             services.AddScoped<ISshService, SshService>();
             services.AddScoped<IUserService, UserService>();
-            services.AddSingleton<IHttpContextAccessor, NullHttpContextAccessor>();
+            services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();

```

Comment:
FWIW microsoft docs say the preferred way is `services.AddHttpContextAccessor();`.